scripts/UpdateSubmodules.ps1

```diff
@@ -50,13 +50,22 @@ try {
     Get-ChildItem "$RepoRoot/modules/*" -Directory | % {
         Push-Location $_
         try {
+            $vcs_name = "BUILD_VCS_NUMBER_" +$_.Name
+            $commit = [environment]::GetEnvironmentVariable($vcs_name)
+
+            if ($commit == $null)
+            {
+                $commit = $(git rev-parse HEAD)
+            }
+
             $data = @{
                 path      = $_
                 module    = $_.Name
-                commit    = $(git rev-parse HEAD)
+                commit    = $commit
```

Comment:
$data.commit is meant to represent the current commit. What we're actualy interested in getting from TC is the newCommit value (See the next loop down)